DataFormats/Detectors/CTP/src/LumiInfo.cxx

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file LumiInfo.cxx
+
+#include "DataFormatsCTP/LumiInfo.h"
+// #include "DataFormatsCTP/Configuration.h"
+
```

Comment:
You forgot the namespace, add `using namespace o2::o2::ctp;`

Reply:
yes, thanks.